README.md

```diff
@@ -2,28 +2,27 @@
 
 [![Step changelog](https://shields.io/github/v/release/bitrise-steplib/steps-git-clone?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/steps-git-clone/releases)
 
-The Step checks out the defined repository state, optionally updates the repository submodules and exports the achieved git repository state properties.
+Checks out the repository, updates submodules and exports git metadata as Step outputs.
 
 <details>
 <summary>Description</summary>
 
-The checkout process depends on the checkout properties: the Step either checks out a repository state defined by a git commit or a git tag, or achieves a merged state of a pull / merge request.
-The Step uses two solutions to achieve the merged state of the pull / merge request: auto merge in the case of a merge branch or diff file (provided by the Git service) and manual merge otherwise.
-Once the desired state is checked out, the Step optionally updates the submodules. In the case of pull / merge request, the Step checks out a detach head and exports the achieved git state properties.
+The checkout process depends on the Step settings and the build trigger parameters (coming from your git server).
 
-### Configuring the Step
+Depending on the conditions, the step can checkout:
+- the merged state of a Pull Request
+- the head of a Pull Request
+- a git tag
+- a specific commit on a branch
+- the head of a branch
+
+The Step also supports more advanced features, such as updating submodules and sparse checkouts.
 
-1. The **Git repository URL** and the ** Clone destination (local)directory path** fields are required fields and are automatically filled out based on your project settings.
-Optionally, you can modify the following fields in the **Clone Config** section:
-1. You can set the **Update the registered submodules?** option to `yes` to pull the most up-to-date version of the submodule from the submodule's repository.
-2. You can set the number of commits you want the Step to fetch in the **Limit fetching to the specified number of commits** option. Make sure you set a decimal number.
+### Configuring the Step
 
-Other **Clone config** inputs are not editable unless you go to the **bitrise.yml** tab, however, to avoid issues, we suggest you to contact our Support team instead.
+The step should work with its default configuration if build triggers and webhooks are set up correctly.
 
-### Troubleshooting
-If you have GitHub Enterprise set up, it works slightly differently on [bitrise.io](https://www.bitrise.io) than on [github.com](https://github.com). You have to manually set the git clone URL, register the SSH key and the webhook.
-If you face network issues in the case of self-hosted git servers, we advise you to contact our Support Team to help you out.
```

Comment:
This is precisely the thing that our support can't help with 😭

Reply:
😅